modal_proto/api.proto

```diff
@@ -2879,6 +2879,70 @@ message VolumeMount {
   bool read_only = 4;
 }
 
+message VolumePutFiles2Request {
+  // Id of volume to put files into.
+  string volume_id = 1;
+
+  // List of files, sorted lexicographically by `path`.
+  repeated File files = 2;
```

Comment:
Do we need the sorting requirement? Are we planning to enforce it?

Reply:
I want to enforce it because it means the caller is aware of the order, which is essential since we refer to files by index in the response. It is not strictly necessary, though.

Reply:
I want to apply the [Robustness Principle](https://en.wikipedia.org/wiki/Robustness_principle) here, that we should suggest a desired format, but be liberal in what we accept. Hence, I think it makes sense to keep this comment but to tolerate unsorted input on the server side.